setup.cfg

```diff
@@ -1,3 +1,2 @@
 [tool:pytest]
-python_paths = .
-norecursedirs = .tox .git */migrations/* */static/* docs venv */{{cookiecutter.project_name}}/*
+testpaths = tests
```

Comment:
Do we need more here? See:
* https://github.com/PublicHealthEngland/nsc-recommendations/blob/develop/setup.cfg#L3-L5
* https://github.com/radiac/django-fastview/blob/develop/setup.cfg#L52-L53

Reply:
Ah yeah, will need adopts, was just to get it working. That's the top level one not the CC one, which has that all in, so won't need the Django bits.

Reply:
Added, I've left coverage off, the test will cover the cookie_cutter so this would only be covering the tests themselves.